### ISSUE TITLE
Can the documentation be updated to state which versions of Biztalk are supported by this tool

### ISSUE THREAD

**Issue by abhijit-accenture** [Azure/aimtool#14]
## Description

I have been doing a feasibility check for Biztalk 2008 to Azure migration. I could not find which versions of Biztalk are supported by this tool for migration.

## Environment and tool versions

- AIM Tool Version (aim --version): 
- Windows Version (systeminfo):
- Shell: (PowerShell | Bash)
- Shell Version ($PSVersionTable | bash --version): 

## Repro steps

Please list out the steps needed to repro the issue, or delete if this is a feature request.

## Expected result

Please describe what you expected to see, or delete if this is a feature request.

## Current result

Please describe what you are seeing, or delete if this is a feature request.

**Comment by probertdaniel** [comment]
Yes, that should certainly be there - for the record, the tool has officially supports BizTalk 2013 R2 and above.
However, it should work on BizTalk 2010 and above with no issues.
Before then, there are changes in the Binding format and some adapter metadata, so you might get errors, although it depends on your applications and what adapters are used.
